kubernetes/resource_kubernetes_node_taint_test.go

```diff
@@ -44,6 +44,35 @@ func TestAccKubernetesResourceNodeTaint_basic(t *testing.T) {
 	})
 }
 
+func TestAccKubernetesResourceNodeTaint_MultipleBasic(t *testing.T) {
+	resourceName := "kubernetes_node_taint.test"
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:          func() { testAccPreCheck(t) },
+		IDRefreshName:     resourceName,
+		ProviderFactories: testAccProviderFactories,
+		CheckDestroy:      testAccKubernetesNodeTaintDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccKubernetesNodeTaintConfig_multipleBasic(),
+				Check: resource.ComposeAggregateTestCheckFunc(
```

Comment:
I'll probably be removing `testAccKubernetesNodeTaintConfig_removeAllTaints()` test step since we need to have minimum 1 taint unless we remove the `minItems` but it feels a bit pointless to have no taints in the node taint resource.